addon/services/infinity.js

```diff
@@ -227,6 +228,7 @@ export default Service.extend({
     }
 
     let initParams = {
+      container: getOwner(this),
```

Comment:
This *seems* like a good idea...but not sure what implications this would have, from a memory- or lifecycle-related perspective.

Reply:
@callovarne What version of ember-infinity are you on?  I had thought this was fixed a while back.  See the below README link.

https://github.com/ember-infinity/ember-infinity#extending-infinitymodel